.dockerignore

```diff
@@ -0,0 +1,12 @@
+build-support/grapl-venv
```

Comment:
Looks like this moves the .dockerignore file, which I think means all other build contexts need updating, right? In order to use it that is, since it's only read from the root of a build context?

Reply:
err nevermind, for some reason i thought i saw src/.dockerignore was removed, but that's not the case.